packages/web-runtime/l10n/Makefile

```diff
@@ -20,7 +20,7 @@ OUTPUT_DIR = .
 TEMPLATE_FILE = ./template.pot
 
 # Available locales for the app.
-LOCALES = bg cs de es fr it nl ko sq sv tr
+LOCALES = bg cs de es fr it nl ko sq sv tr zh-Hans
```

Comment:
Zh not possible ?

Reply:
The language key on transifex.com is `zh-Hans`, apparently `zh` is not that commonly used...